src/sizzle.js

```diff
@@ -1226,10 +1280,16 @@ Expr = Sizzle.selectors = {
 			var pattern = classCache[ className + " " ];
 
 			return pattern ||
-				(pattern = new RegExp( "(^|" + whitespace + ")" + className + "(" + whitespace + "|$)" )) &&
-				classCache( className, function( elem ) {
-					return pattern.test( typeof elem.className === "string" && elem.className || typeof elem.getAttribute !== "undefined" && elem.getAttribute("class") || "" );
-				});
+				( pattern = new RegExp( "(^|" + whitespace +
+					")" + className + "(" + whitespace + "|$)" ) ) && classCache(
+						className, function( elem ) {
+							return pattern.test(
+								typeof elem.className === "string" &&
+								elem.className || typeof elem.getAttribute !== "undefined" &&
```

Comment:
Please move `elem.className ||` to the previous line so that it's clearer what is grouped with what (as `&&` takes precedence before `||`).

The same remark to the line below - move `elem.getAttribute( "class" ) ||` to this line.

Reply:
Do yo think it looks better in this way? 

![PR-improvement](https://user-images.githubusercontent.com/1837650/58383326-9a992480-7fd5-11e9-92d4-1fe052f74df4.png)

Reply:
@JuanMaRuiz In this way to break the second part of the expression:
```js
typeof elem.getAttribute !== undefined && elem.getAttribute( "class" )
```
is broken into two lines and the second part is put in the same line as the third condition: `""`. That's confusing to me. As I wrote above, I'd prefer each part to be kept in one line.

Reply:
If it's not possible to fit in the `elem.getAttribute( "class" )` part in the previous line, I'd move the last `""` to a new line to separate it from the previous condition. In that case, the line with `elem.getAttribute( "class" )` should be indented more:
```js
								typeof elem.className === "string" && elem.className ||
								typeof elem.getAttribute !== "undefined" &&
									elem.getAttribute( "class" ) ||
								""
```

Reply:
Done